### PR TITLE
Eth coaching playbook

### PR DESCRIPTION
**Pull request by chair28980**
This playbook aims to "coach the coach" aka instruct a person how to guide a new Ethereum user through the process of navigating transactions and the potential issues they may face.

This is a first "final" draft and I am seeking review!

[HackMD Draft - with Coach](https://hackmd.io/1id4VIkARJ-RXi1WqRUyVw)
[HackMD Draft - w/o Coach](https://hackmd.io/u7d_ovMNTNCAt0XoI6RYpQ)

### CODE REVIEW

**Review by luxumbra — CHANGES_REQUESTED** [review]
Hey @chair28980 . Thanks for the PR. Looks like you need to copy the contents of `coaching-eth.md` into the empty `how-to-coach-eth.md`. 😄

**Comment by chair28980** [comment]
> Hey @chair28980 . Thanks for the PR. Looks like you need to copy the contents of `coaching-eth.md` into the empty `how-to-coach-eth.md`. 😄

lulz, thank you

**Review by luxumbra — CHANGES_REQUESTED** [review]
@chair28980 once you've got your final draft, can you add the copy to the `how-to-coach-eth.md` file as it's currently showing an empty page. Also, for consistency, in the `sidebars.js` file could you add the title emoji to the new entry there. 🙏🏻 😃

**Comment by luxumbra** [comment]
> @chair28980 once you've got your final draft, can you add the copy to the `how-to-coach-eth.md` file as it's currently showing an empty page. ~Also, for consistency, in the `sidebars.js` file could you add the title emoji to the new entry there~. 🙏🏻

Sorry, just seen your reply to my otrher comment. 🤦🏻

**Comment by chair28980** [comment]
> > @chair28980 once you've got your final draft, can you add the copy to the `how-to-coach-eth.md` file as it's currently showing an empty page. ~Also, for consistency, in the `sidebars.js` file could you add the title emoji to the new entry there~. 🙏🏻
> 
> Sorry, just seen your reply to my otrher comment. 🤦🏻

Nooo worries. I'm going to add a step covering canceling a tx and the "user without a coach" playbook asap.

File: sidebars.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ module.exports = {
     "📚 Playbooks": [
       "playbooks/browse",
       "playbooks/how-to-make-it-without-technical-skills",
-      "playbooks/coaching-eth",
+      "playbooks/how-to-coach-eth",
```
Comment (luxumbra) [inline sidebars.js:47]: @chair28980 I just checked your last update and all good but I noticed there's a conflict in `sidebars.js`. once that's sorted, it's good to go. 🙏🏻

**Review by luxumbra — CHANGES_REQUESTED** [review]

File: sidebars.js (hunk 1 above)
Comment (chair28980) [reply sidebars.js:47]: GAHH WHY IS THIS HAPPENING AHHHH

Tyty will get things fixed and ping when ready.

Comment (chair28980) [reply sidebars.js:47]: GAHH WHY IS THIS HAPPENING AHHHH

Tyty will get things fixed and ping when ready.

Comment (luxumbra) [reply sidebars.js:47]: > GAHH WHY IS THIS HAPPENING AHHHH
😅 I can relate. I responded to your DM on Discord.

**Comment by chair28980** [comment]
@luxumbra ok, im more confused now, i deleted the offending line but alas the conflict persists

**Review by luxumbra — APPROVED** [review]

**Pull request merged**